### PR TITLE
CLOUDP-126257: Create command 'atlas backups exports buckets describe'

### PR DESCRIPTION
**Pull request by Razeer123**
## Proposed changes

I added the "atlas backups exports buckets describe" command to the CLI and a unit test to verify the correctness of the implementation.

_Jira ticket:_ [CLOUDP-126257](https://jira.mongodb.org/browse/CLOUDP-126257)

## Command syntax and output

```
michal.derej@M-L7H6RDJWQX mongodb-atlas-cli % ./bin/atlas backup exports buckets describe --bucketId 62c569f85b7a381c093cc539 -P dev
ID                         BUCKET NAME          CLOUD PROVIDER   IAM ROLE ID
62c569f85b7a381c093cc539   michal-test-bucket   AWS              62c5604822125b72fac3f9b6
```

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/CONTRIBUTING.md) (if appropriate)
- [x] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [x] I have updated [test/README.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/README.md) (if an e2e test has been added)
- [x] I have run `make fmt` and formatted my code

### CODE REVIEW

File: docs/atlascli/command/atlas-backups-exports-buckets-describe.txt [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Retrieve one snapshot export bucket.
```
Comment (sarahsimpers) [inline docs/atlascli/command/atlas-backups-exports-buckets-describe.txt:15]: Nit: changing for consistency with the language we use now for API
```suggestion
Return one snapshot export bucket.
```

File: docs/atlascli/command/atlas-backups-exports-buckets-describe.txt [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+     - string
+     - false
+     - Profile to use from your configuration file.
+
+Examples
+--------
+
+.. code-block::
+
+   The following example retrieves the continuous backup export bucket specified by ID:
```
Comment (sarahsimpers) [inline docs/atlascli/command/atlas-backups-exports-buckets-describe.txt:73]:
```suggestion
   The following example returns the continuous backup export bucket specified by the ID:
```

**Review by sarahsimpers — CHANGES_REQUESTED** [review]
Just a few small changes, thanks!

File: docs/atlascli/command/atlas-backups-exports-buckets-describe.txt [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - --bucketId
+     - string
+     - true
+     - Unique identifier of the bucket
```
[the remark this answers is not in the document]
Comment (Razeer123) [reply docs/atlascli/command/atlas-backups-exports-buckets-describe.txt:38]: It's assigned by Atlas.

**Review by sarahsimpers — APPROVED** [review]

**Review by fmenezes — APPROVED** [review]
LGTM

**Pull request merged**